### DISCUSSION TITLE
Zeiss dicom removal?

### DISCUSSION THREAD

**Discussion by brendcurran** [marksgraham/OCT-Converter#110] [General]
Hi there! Really appreciate the work you've done here--it's been invaluable for me and my company.

I'm curious, I see that you added Zeiss dicom support in November, but removed the reader from the library three days ago. Just curious if that's temporary, or if support for Zeiss dicom images will be going away forever? Thanks!

**Comment by marksgraham** [comment]
Hi, I'm glad you find the repository useful! I'm afraid support for this is gone, Zeiss requested that I remove it.
